components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/IdentityRecoveryConstants.java

```diff
@@ -59,26 +59,34 @@
     // Notification channel claims.
     public static final String VERIFY_EMAIL_CLIAM = "http://wso2.org/claims/identity/verifyEmail";
     public static final String EMAIL_VERIFIED_CLAIM = "http://wso2.org/claims/identity/emailVerified";
+    public static final String EMAIL_ADDRESS_PENDING_VALUE_CLAIM =
+            "http://wso2.org/claims/identity/emailaddress.pendingValue";
+    public static final String MOBILE_VERIFIED_CLAIM = "http://wso2.org/claims/identity/phoneVerified";
     public static final String PREFERRED_CHANNEL_CLAIM = "http://wso2.org/claims/identity/preferredChannel";
 
     public static final String ASK_PASSWORD_CLAIM = "http://wso2.org/claims/identity/askPassword";
     public static final String ADMIN_FORCED_PASSWORD_RESET_CLAIM = "http://wso2.org/claims/identity/adminForcedPasswordReset";
     public static final String OTP_PASSWORD_CLAIM = "http://wso2.org/claims/oneTimePassword";
     public static final String USER_ROLES_CLAIM = "http://wso2.org/claims/role";
+    public static final String EMAIL_ADDRESS_CLAIM = "http://wso2.org/claims/emailaddress";
```

Comment:
Can we use `NotificationChannels. EMAIL_CHANNEL.getClaimUri()` ? WDYT?

Reply:
Since this is not regarding the notification channels, hence using this claim in fine. Therefore resolving the comment.